### ISSUE TITLE
MacOS installers are wrong (MacOS, platform GCP tested). Some of them are built on the same commit

### ISSUE THREAD

**Issue by dperezro** [openshift/okd#376]
**Describe the bug**
At least the last 4 MacOS installers located [here](https://github.com/openshift/okd/releases) are wrongly build. Some of them are built on the same commit (`63200c80c431b8dbaa06c0cc13282d819bd7e5f8`), as you can see when doing `openshift-install version`:

```
~/Downloads » ./openshift-install-mac-4.5.0-0.okd-2020-10-15-235428/openshift-install version
./openshift-install-mac-4.5.0-0.okd-2020-10-15-235428/openshift-install 4.5.0-0.okd-2020-10-15-235428
built from commit 63200c80c431b8dbaa06c0cc13282d819bd7e5f8
release image quay.io/openshift/okd@sha256:67cc7cb47d56237adcf0ecc2ee76446785add5fa236cd08746b55f578b9200a5

~/Downloads » ./openshift-install-mac-4.5.0-0.okd-2020-10-03-012432/openshift-install version
./openshift-install-mac-4.5.0-0.okd-2020-10-03-012432/openshift-install 4.5.0-0.okd-2020-10-03-012432
built from commit 63200c80c431b8dbaa06c0cc13282d819bd7e5f8
release image quay.io/openshift/okd@sha256:ae9949b075844ccaeb149dd0a33ad380976762e5cdeb4735d99063c2a2178401

~/Downloads » ./openshift-install-mac-4.5.0-0.okd-2020-09-18-202631/openshift-install version
./openshift-install-mac-4.5.0-0.okd-2020-09-18-202631/openshift-install 4.5.0-0.okd-2020-09-18-202631
built from commit 63200c80c431b8dbaa06c0cc13282d819bd7e5f8
release image quay.io/openshift/okd@sha256:5fd1fe9707a9a4f53c8ccafad0cf44824a3a0b51e197f3fbc98d0884a9ddcf4f

~/Downloads » ./openshift-install-mac-4.5.0-0.okd-2020-09-04-180756/openshift-install version
./openshift-install-mac-4.5.0-0.okd-2020-09-04-180756/openshift-install 4.5.0-0.okd-2020-09-04-180756
built from commit 63200c80c431b8dbaa06c0cc13282d819bd7e5f8
release image quay.io/openshift/okd@sha256:eeb7ba7c0ca5749f2e27e0951da70263658301f5bfa4fdd86524d73bfdeb7cac

~/Downloads » ./openshift-install-mac-4.5.0-0.okd-2020-08-12-020541/openshift-install version
./openshift-install-mac-4.5.0-0.okd-2020-08-12-020541/openshift-install 4.5.0-0.okd-2020-08-12-020541
built from commit 699277bb61706731d687b9e40700ebf4630b0851
release image quay.io/openshift/okd@sha256:6974c414be62aee4fde24fe47ccfff97c2854ddc37eb196f3f3bcda2fdec17b4
```

I've tested this using GCP only and a MacOS installer. Issue creates another problem: image `fedora-coreos-32-20200629-3-0-gcp-x86-64.tar.gz` [doesn't exist in GCP anymore](https://storage.googleapis.com/fedora-coreos-cloud-image-uploads). Commit `63200c80c431b8dbaa06c0cc13282d819bd7e5f8` seen [here](https://github.com/openshift/installer/blob/63200c80c431b8dbaa06c0cc13282d819bd7e5f8/data/data/fcos-amd64.json#L74) clearly shows that it's using such image and new OKD installations using the MacOS installer are not working at all.

**Version**
4.5.0-0.okd-2020-10-15-235428
4.5.0-0.okd-2020-10-03-012432
4.5.0-0.okd-2020-09-18-202631
4.5.0-0.okd-2020-09-04-180756

**How reproducible**
100% of the times.

**Log bundle**
An example of configuration used:
```
apiVersion: v1
baseDomain: okd_example.domain.com
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform: 
    gcp:
      type: n1-standard-16
  replicas: 10
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform: 
    gcp:
      type: n1-standard-16
  replicas: 3
metadata:
  creationTimestamp: null
  name: okd_example
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 10.0.0.0/16
  networkType: OVNKubernetes
  serviceNetwork:
  - 172.30.0.0/16
platform:
  gcp:
    projectID: gcp_project_example
    region: europe-west1
publish: External
pullSecret: '{some_pull_secret}'
sshKey: |
  ssh-rsa ANY_PUBLIC_KEY
```

**Comment by vrutkovs** [comment]
>the last 4 MacOS installers located here are wrongly build

What does it mean?

> Some of them are built on the same commit (63200c80c431b8dbaa06c0cc13282d819bd7e5f8),

This expected. 4.5 OKD installer is effectively frozen, we're working on OKD 4.6 in `fcos` branch

>  Issue creates another problem: image fedora-coreos-32-20200629-3-0-gcp-x86-64.tar.gz doesn't exist in GCP anymore.

*sigh* Right, we'll need @dustymabe to re-upload it. Luckily OKD 4.6 installer is using fresh FCOS version

**Comment by dperezro** [comment]
> > Some of them are built on the same commit (63200c80c431b8dbaa06c0cc13282d819bd7e5f8),
> 
> This is expected. 4.5 OKD installer is effectively frozen, we're working on OKD 4.6 in `fcos` branch
> 
> > Issue creates another problem: image fedora-coreos-32-20200629-3-0-gcp-x86-64.tar.gz doesn't exist in GCP anymore.
> 
> _sigh_ Right, we'll need @dustymabe to re-upload it. Luckily OKD 4.6 installer is using fresh FCOS version

@vrutkovs @dustymabe thank you both!

**Comment by dperezro** [comment]
Hi, any luck with this one? @vrutkovs @dustymabe ?

**Comment by vrutkovs** [comment]
Fixed in [OKD 4.6](https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.6.0-0.okd-2020-11-27-200126):
* installer built from https://github.com/openshift/installer/tree/fcos
* fresh FCOS GCP image is used
